product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx

```diff
@@ -4,58 +4,59 @@ navTitle: "Version 5.1.0"
+| BDR       | 5.1.0   | Feature         | Added group name in the bdr.event_summary view. |
+| BDR       | 5.1.0   | Feature         | Added SLES support. SLES 15sp4 is now supported. |
+| BDR       | 5.1.0   | Feature         | Added subscription status column to the group subscription summary view. This feature allows you to distinguish whether NULL values are due to a node being down or a subscription being disabled.|
+| BDR       | 5.1.0   | Feature         | Added NOT group filter to Group Commit. This feature allows you to invert the meaning of a group filter to include all nodes except the ones in specified groups.|
+| BDR       | 5.1.0   | Feature         | Added `bdr.drop_node_group`. You can now drop empty node groups.|
+| BDR       | 5.1.0   | Feature         | Added SYNCHRONOUS_COMMIT to Commit Scopes. This feature allows dynamic synchronous_commit-like behavior for replication.|
+| BDR       | 5.1.0   | Feature         | Added event_node_name column to `bdr.event_summary`. |
+| BDR       | 5.1.0   | Feature         | Added write leader election to event history. Added information about the node that's elected as a write leader for each group in the event_history catalog. Also improved the reporting of raft instance ids in the event_detail of event_history.|
+| BDR       | 5.1.0   | Feature         | Added ability to allow exporting and importing of other Raft instance snapshots. This feature allows exporting and importing snapshots for other instances and not only top Raft instances.|
+| BDR       | 5.1.0   | Bug fix         | Fixed memory leak in consensus process.  |
+| BDR       | 5.1.0   | Bug fix         | Fixed issue where a node can be inconsistent with the group after rejoining. If a node was part of a subgroup, parted, and then rejoined to the group, it could be inconsistent with the group. The changes from some nodes of the group were replayed from a wrong starting point, resulting in potential data loss. |
+| BDR       | 5.1.0   | Bug fix         | Fixed join and replication when SDW and standby_slot_names are set. |
+| BDR       | 5.1.0   | Bug fix         | Fixed spurious warnings when processing sequence options. |
+| BDR       | 5.1.0   | Bug fix         | Fixed upgrades for nodes with CRDTs. |
+| BDR       | 5.1.0   | Bug fix         | Adjusted lag tracking parameters for LCRs from pg_stat_replication. |
+| BDR       | 5.1.0   | Bug fix         | Adjusted node routing options defaults based on node kind. This change is related only to the display of the information and not the behavior. For example, witness nodes aren't marked as candidates for receiving writes. |
+| BDR       | 5.1.0   | Bug fix         | All sequences are now converted to distributed during create node. |
+| BDR       | 5.1.0   | Bug fix         | Fixed streaming transactions with `standby_slot_names`. This might have led to a subscriber-only node getting ahead of a data node. |
```

Comment:
```suggestion
| BDR       | 5.1.0   | Bug fix         | Fixed streaming transactions with `standby_slot_names`.<br/>This might have led to a subscriber-only node getting ahead of a data node. |
```